control-plane/grpc/src/operations/volume/traits.rs

```diff
@@ -1818,29 +1818,25 @@ impl Event for Volume {
             Some(target) => target.node().to_string(),
             None => "".to_string(),
         };
-        let event_meta = EventMeta::new();
         let event_source = EventSource::new(node_id);
         EventMessage {
             category: EventCategory::Volume as i32,
             action: EventAction::Create as i32,
             target: self.uuid().to_string(),
-            metadata: Some(EventMeta {
-                source: Some(event_source),
-                ..event_meta
-            }),
+            metadata: Some(EventMeta::from_source(event_source)),
         }
     }
 }
 
 // Create volume delete event message.
 impl Event for ValidatedDestroyVolumeRequest {
     fn event(&self) -> EventMessage {
-        let event_meta = EventMeta::new();
+        let event_source = EventSource::new("".to_string());
```

Comment:
What is `""` here?

Reply:
I couldn't get the node name from "ValidatedDestroyVolumeRequest". So I was passing empty string as the node name while creating source.

Reply:
Right, I guess depends what we want node_name here to signify..
If we want it to signify the node_name of the running pod perhaps we can, maybe worth a backlog item but for now lgtm.

Reply:
Yes. I think it should signify the node where the volume is created. Will add it to TODO.